### PR TITLE
Prepare for move to flutter/packages

### PR DESCRIPTION
**Pull request by bryanoltman**
This PR contains changes made in https://github.com/flutter/packages/pull/2596 to satisfy the stricter formatting and analysis options in that repo. This does not contain the rename of `Scene` to `StagerScene`, as that issue is currently unresolved. This should not contain any functionality changes.

### CODE REVIEW

File: example/lib/pages/posts_list/posts_list_page_scenes.dart [hunk 1]
```diff
@@ -54,25 +62,29 @@ class WithPostsScene extends BasePostsListScene {
...
+/// A Scene showing the [PostsListPage] in a loading state.
 class LoadingScene extends BasePostsListScene {
   @override
   String get title => 'Loading';
 
   @override
   Future<void> setUp() async {
     await super.setUp();
-    final completer = Completer<List<Post>>();
+    final Completer<List<Post>> completer = Completer<List<Post>>();
```
Comment (jacobsimionato) [inline example/lib/pages/posts_list/posts_list_page_scenes.dart:73]: It's surprising to me that this is necessary...

File: example/test/pages/post_detail_page_test.dart [hunk 2]
```diff
@@ -1,33 +1,34 @@
-import 'package:example/pages/posts_list/posts_list_page_scenes.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:stager/stager.dart';
+import 'package:stager_example/pages/posts_list/posts_list_page_scenes.dart';
 
 void main() {
   testWidgets('shows a loading state', (WidgetTester tester) async {
-    final scene = LoadingScene();
+    final Scene scene = LoadingScene();
```
Comment (jacobsimionato) [inline example/test/pages/post_detail_page_test.dart:8]: Why is this necessary?

**Review by jacobsimionato — APPROVED** [review]
Looks good! Wow that's the first time I've reviewed Linux app C code. I just had a couple of questions about the specific type declarations.

File: example/lib/pages/posts_list/posts_list_page_scenes.dart (hunk 1 above)
Comment (bryanoltman) [reply example/lib/pages/posts_list/posts_list_page_scenes.dart:73]: Yup, :( https://dart-lang.github.io/linter/lints/always_specify_types.html

File: example/test/pages/post_detail_page_test.dart (hunk 2 above)
Comment (bryanoltman) [reply example/test/pages/post_detail_page_test.dart:8]: https://dart-lang.github.io/linter/lints/always_specify_types.html
